src/analysis/special_functions/trigonometric.lean

```diff
@@ -1748,15 +1750,130 @@ by simp only [tan_eq_sin_div_cos]; exact
 lemma continuous_on_tan : continuous_on tan {x | cos x ≠ 0} :=
 by { rw continuous_on_iff_continuous_restrict, convert continuous_tan }
 
-lemma has_deriv_at_tan_of_mem_Ioo {x : ℝ} (h : x ∈ set.Ioo (-(π/2):ℝ) (π/2)) :
+lemma has_deriv_at_tan_of_mem_Ioo {x : ℝ} (h : x ∈ Ioo (-(π/2):ℝ) (π/2)) :
   has_deriv_at tan (1 / (cos x)^2) x :=
 has_deriv_at_tan (cos_ne_zero_iff.mp (ne_of_gt (cos_pos_of_mem_Ioo h.1 h.2)))
 
-lemma differentiable_at_tan_of_mem_Ioo {x : ℝ} (h : x ∈ set.Ioo (-(π/2):ℝ) (π/2)) :
+lemma differentiable_at_tan_of_mem_Ioo {x : ℝ} (h : x ∈ Ioo (-(π/2):ℝ) (π/2)) :
   differentiable_at ℝ tan x :=
 (has_deriv_at_tan_of_mem_Ioo h).differentiable_at
 
-lemma deriv_tan_of_mem_Ioo {x : ℝ} (h : x ∈ set.Ioo (-(π/2):ℝ) (π/2)) : deriv tan x = 1 / (cos x)^2 :=
+lemma deriv_tan_of_mem_Ioo {x : ℝ} (h : x ∈ Ioo (-(π/2):ℝ) (π/2)) : deriv tan x = 1 / (cos x)^2 :=
 (has_deriv_at_tan_of_mem_Ioo h).deriv
 
+lemma continuous_on_tan_of_mem_Ioo : continuous_on tan (Ioo (-(π/2)) (π/2)) :=
+begin
+  refine continuous_on_tan.mono _,
+  intros x hx,
+  simp only [set.mem_set_of_eq],
+  exact ne_of_gt (cos_pos_of_mem_Ioo hx.1 hx.2),
+end
+
+open filter
+open_locale topological_space
+
+lemma tendsto_sin_pi_div_two : tendsto sin (𝓝[Iio (π/2)] (π/2)) (𝓝 1) :=
+begin
+  rw show (𝓝 1 = 𝓝 (sin (π/2))), by rw sin_pi_div_two,
+  exact continuous_sin.continuous_within_at
```

Comment:
I'm not familiar at all with `show`. Is it considered "better" than using `convert`? For example, I can golf this proof down to `convert continuous_sin.continuous_within_at, simp,` - would that be less preferable than using `show`? (The same applies for the other proofs.)

Reply:
I personally find the `convert` version easier to read, but again - I'm not familiar with the benefits of `show`.

Reply:
I introduced that because you had to provide implicit arguments, but `convert` + `simp` is fine. It's probably slower but probably not an issue.

Reply:
Okay thank you!